apps/artwork/components/artists/test/fixture.coffee

```diff
@@ -0,0 +1,72 @@
+{ fabricate } = require 'antigravity'
+
```

Comment:
Another approach could be to combine the two files and export two different variables. I.E.

```
module.exports.artistsWithoutExhibitions = <contents>
module.exports.artistsWithExhibitions = <contents>
```
and
```
{ artistsWithoutExhibitions, artistsWithExhibitions } = require 'fixture'
```